### ISSUE TITLE
Optional: only allow 1 instance of Nemo at a time

### ISSUE THREAD

**Issue by Dan-cer** [linuxmint/nemo#3054]
```
 * Nemo version 5.2.4 and 5.3.4
 * Issue with windowed nemo
 * Mint 20.3 (Device-1) and 21 Cinnamon (Device-2)
 * Device-1: NVIDIA GP108 [GeForce GT 1030] driver: nvidia v: 515.65.01
 * Device-2: Intel CoffeeLake-S GT2 [UHD Graphics 630] vendor: Dell CometLake-S driver: i915
 * 64 bit
 ```

**Issue**
feature request
With each file operation a new instance of Nemo is starting. Instead I want to have an option to allow only 1 instance at a time.

**Steps to reproduce**
Clicking on downloaded file 1 opens Nemo 1.
Clicking on downloaded file 2 opens Nemo 2.
and so on.

**Expected behaviour**
Clicking on downloaded file 2 opens either a new tab within Nemo 1 or highlights file 2 where it is saved.

**Other information**
This option to allow only one instance at a time is provided in some other file managers like Double Commander.

**Comment by Jeremy7701** [comment]
Surely double clicking a downloaded file runs the default program for that file type?
Nemo is the default program to run if your file type should be a folder...

**Comment by Dan-cer** [comment]
Yes I'm sure. Because I'm not clicking on the file directly but on the folder icon within the download manager (of the browser). That opens the destinated folder directly and I can execute or open that downloaded file.

**Comment by Jeremy7701** [comment]
2 possible "solutions" :-

1. Open your download folder (with nemo) and leave it open, or
2. [My favourite] Change your default download destination to 'Desktop' as most downloaded files are either soon deleted or should be moved to a more permanent location. Indeed, I don't have a 'Downloads' folder at all.

**Comment by Dan-cer** [comment]
The reason for my feature request is that just when I have Nemo already opened (minimized), a second instance is opened. Your 1st solution therefore does not apply here. On the contrary. When Nemo is closed, opening the download folder makes sense and is desired. 
Your 2nd solution makes no difference to me. My desire to allow only 1 instance of Nemo at a time is not only justified in the download scenario. Also when I connect a USB stick or hard drive, I want the corresponding directory to open in Nemo. This option is available, but, if checked, it opens another instance of Nemo. So far, I can only prevent this by unchecking "Automatically open a new folder for mounted devices".

My question straight out: Is it very costly to create an additional setting option to allow Nemo only in 1 instance?

**Comment by morfikov** [comment]
This is a real problem when you call nemo from other apps, for instance qbittorrent -- it will spawn a new nemo window for each opened folder... I wanted to switch to nemo, but I stay with spacefm, because it will just add a new tab with the open folder, believe me we don't live in the early 90' and we certainly don't need the IE5 behavior anymore... :]

**Comment by Dan-cer** [comment]
In [this discussion on Github](https://github.com/orgs/linuxmint/discussions/351) I found a solution that in most cases works:
> There's a way this can be achieved for now (in LM 21.3, LMDE 6, or greater):
> 1. On the default menu write `nemo`
> 2. Right-click on the nemo file manager application ("Files")
> 3. A submenu should open, click on the 'properties' option
> 4. It will open a small window, and where it is Command: add `nemo --existing-window %U` instead of the default text that is there (nemo %U)
> 5. Then click accept to conclude the change

> If you want to revert this follow all steps from 1 to 3 and put `nemo %U` back in the Command input.

[If you then want to open Nemo always with the same position:
In "Windows" settings open the tab "Behavior" and in the second section set "Position of new opened window" to cursor or manually.]
